lua/entities/gmod_tardis/modules/sh_cloak.lua

```diff
@@ -50,7 +50,7 @@ if SERVER then
 
     ENT:AddHook("HandleE2", "cloak", function(self,name,e2)
         if name == "Phase" and TARDIS:CheckPP(e2.player, self) then
-            return self:ToggleCloak() and 1 or 0
+                return (self:GetData("power-state", false) and self:ToggleCloak()) and 1 or 0
```

Comment:
You should use `self:GetPower()` here instead of checking the power state directly